packages/paste-core/components/button/src/styles.ts

```diff
@@ -115,20 +113,26 @@ const variantPrimaryEnabled = (props: ButtonWrapperProps): SerializedStyles =>
     baseEnabled(props),
     variantPrimaryBase(props),
     css`
-      border-color: ${themeGet('borderColors.colorBorderPrimary')(props)};
       background-color: ${themeGet('backgroundColors.colorBackgroundPrimary')(props)};
+      box-shadow: inset 0 0 0 ${themeGet('borderWidths.borderWidth20')(props)}
+        ${themeGet('backgroundColors.colorBackgroundPrimary')(props)};
 
       &:hover {
-        border-color: ${themeGet('borderColors.colorBorderPrimaryDarker')(props)};
         background-color: ${themeGet('backgroundColors.colorBackgroundPrimaryDarker')(props)};
+        box-shadow: inset 0 0 0 ${themeGet('borderWidths.borderWidth20')(props)}
+          ${themeGet('backgroundColors.colorBackgroundPrimaryDarker')(props)};
       }
       &:focus {
-        border-color: ${themeGet('borderColors.colorBorderPrimaryDarker')(props)};
         background-color: ${themeGet('backgroundColors.colorBackgroundPrimary')(props)};
+        box-shadow: inset 0 0 0 ${themeGet('borderWidths.borderWidth20')(props)}
+            ${themeGet('backgroundColors.colorBackgroundPrimaryDarker')(props)},
+          ${themeGet('shadows.shadowFocus')(props)};
       }
       &:active {
-        border-color: ${themeGet('borderColors.colorBorderPrimaryDarker')(props)};
         background-color: ${themeGet('backgroundColors.colorBackgroundPrimaryDark')(props)};
+        box-shadow: inset 0 0 0 ${themeGet('borderWidths.borderWidth20')(props)}
+            ${themeGet('backgroundColors.colorBackgroundPrimaryDarker')(props)},
+          ${themeGet('shadows.shadowFocus')(props)};
```

Comment:
Is there a better way to do this, since we can't set the individual parameters of `box-shadow` separately?